core/element_unary_operations.h

```diff
@@ -7,6 +7,7 @@
 
 #include <cmath>
 
+#include "arg_list.h"
```

Comment:
unused?

Reply:
No. It's common to this and the new `element_trigonometry_operations.h`